### PR TITLE
Add banner option to simply insert comments at top of bundle

### PR DESCRIPTION
**Pull request by fjc0k**
:see_no_evil:请签收一枚迷弟的第一个PR~

This PR adds banner option to simply insert comments at top of bundle.

### CODE REVIEW

**Comment by egoist** [comment]
No need to use `@preserve` keyword, once `banner` is set it should always be inserted.

File: package.json [hunk 1]
```diff
@@ -15,6 +15,8 @@
   "scripts": {
     "test": "BABEL_ENV=test jest --env node && xo src/*.js && npm run build",
     "test:cov": "BABEL_ENV=test jest --env node --coverage && xo src/*.js",
+    "wintest": "set BABEL_ENV=test && jest --env node && xo src/*.js && npm run build",
```
Comment (egoist) [inline package.json]: let's use `cross-env` instead

**Comment by fjc0k** [comment]
When `compress` is set to `true`, for UMD minified bundle, `rollup-plugin-uglify` will remove banner (See: [https://github.com/TrySound/rollup-plugin-uglify/issues/7](https://github.com/TrySound/rollup-plugin-uglify/issues/7)). So, we need `@preserve` keyword to tell Uglify to keep it.

But now, I find a new way to keep banner without `@preserve` keyword (See: [https://github.com/mishoo/UglifyJS2#output-options](https://github.com/mishoo/UglifyJS2#output-options)):
```js
    plugins.push(require('rollup-plugin-uglify')({
      output: {
        // Preserve banner
        preamble: banner
      }
    }))
```

Thx for your reviews.😆

**Comment by egoist** [comment]
I think I need to refactor the test a bit but in a separate PR, seems good to merge though 👍🏻

**Comment by fjc0k** [comment]
I have learned a lot from the first PR. Thanks!

笔芯~:heart:

**Comment by egoist** [comment]
Released in `v0.16.0-rc.1`

**Comment by jitendersandhu** [comment]
Thanks for adding a simpler option to preserve banner text. 👍🏻

**Pull request merged**